assets/js/app.js

```diff
@@ -49,12 +49,21 @@ Hooks.Items = {
     })
 
     this.el.addEventListener("update-indexes", e => {
-        console.log('yyyyy')
         const ids = [...document.querySelectorAll(".item")].map( i => i.dataset.id)
         hook.pushEventTo("#items", "updateIndexes", {ids: ids})
     })
   }
 }
+
```

Comment:
I'd want an in-line comment to where I can read more about how `Hooks` work. 
```suggestion
// See: hexdocs.pm/phoenix_live_view/js-interop.html#client-hooks-via-phx-hook
Hooks.Counter = {
```